### DISCUSSION TITLE
Having trouble getting this to run on linux

### DISCUSSION THREAD

**Discussion by Jrnyman** [sultim-t/xash-rt#114] [Q&A]
Yo I saw some folks run this on linux but I just cant it to work for me, I guess I'm just doing something wrong.

What I know is I need VCRun2019 installed, I ran it without it installed via winetricks and this error comes up:
![image](https://user-images.githubusercontent.com/37476294/222574946-374ea1c9-9537-48e4-ac77-a30c9e3cc82f.png)

After I install vcrun2019 and run it, I get this error instead: 
![image](https://user-images.githubusercontent.com/37476294/222575411-24518870-2b5c-4fe2-81d3-bf73955e656d.png)
and pressing "retry" takes me to this: 
![image](https://user-images.githubusercontent.com/37476294/222575496-a0c9caec-2d99-4234-8f08-5b96de34bbf6.png)

So if there's anything I can do to make it work I'd really appreciate the help, thanks
_(I'm using Nobara Linux 37, and if I need to dump any logs tell me where to find them and I'll hand them over)_

**Comment by Toughbook-CFC2** [comment]
I was stuck at the second error you posted for a while as well. I set up all parameters in lutris as was shown in the discussions here.
Now I'm stuck at a different error. It still feels like progress though.
Install lutris
add a game with the plus in the upper left corner, 'add locally installed...'
set up the name (HL1RT or something), select Wine as runner
under game options add a path to xash3d.exe, maybe select a different wineprefix
under runner options I selected 'lutris-7.2-2-x86_64' 
under system options I added the environment variable 'RADV_PERFTEST' 'rt,emulate_rt' as a key value pair because I have an older AMD GPU

Enjoy the beauty of a different error message! ;D
